geode-core/src/main/java/org/apache/geode/internal/cache/locks/TXOriginatorRecoveryProcessor.java

```diff
@@ -74,19 +70,17 @@ static void sendMessage(Set members, InternalDistributedMember originator, TXLoc
     }
 
     // keep waiting even if interrupted
-    // for() loop removed for bug 36983 - you can't loop on waitForReplies()
     dm.getCancelCriterion().checkCancelInProgress(null);
     try {
       processor.waitForRepliesUninterruptibly();
     } catch (ReplyException e) {
       e.handleCause();
     }
 
-    // release txLockId...
```

Comment:
In this block of code, I think it would be clearer to remove everything (including the false logging statement) or just leave as is.

Reply:
Can you be a little more specific?  I'm unsure which block of code you are talking about? In github reviews, you can now highlight multiple lines for comment, that would help me out in determining the block of code.  I think you're referring to lines 80-82 

```java
if (logger.isDebugEnabled()) {
  logger.debug("TXOriginatorRecoveryProcessor releasing: {}", txLockId);
}
```